### DISCUSSION TITLE
Mutability of struct cross languages

### DISCUSSION THREAD

**Discussion by Xyncgas** [fsharp/fslang-suggestions#1279] [General]
If I made a struct in C#, with member to mutate it
Now in F# I create the struct and I call that member
The question is, in this scenario, is it mutable without me using ```mutable``` keyword, in F# standard, for one I feel conflicted

**Comment by realparadyne** [comment]
Yes it is.

**Reply by Xyncgas** [reply]
So you support the idea it can be mutated or you are saying it actually mutates the struct
Because it doesn't for me, it might be a bug or not it's weird

**Reply by realparadyne** [reply]
I'm saying I tested it out of interest, and the member function can mutate fields in the struct.  The struct, its definition and code for members exist outside of F#, it would not be really possible for the F# compiler to analyse the IL of a library it is referencing to determine that what a member does internally is somehow not allowed.  Then we would be in real trouble trying to use all of the .Net libraries!
